cli/src/commands/runTests.js

```diff
@@ -47,7 +47,7 @@ type TestGroup = {
  * directory.
  */
 const basePathRegex = new RegExp(
-  `definitions${path.sep}npm${path.sep}(\@[^${path.sep}]*${path.sep})?[^${path.sep}]*${path.sep}?`,
+  'definitions/npm/(\@[^/]*/)?[^/]*/?',
```

Comment:
Place everything on one line to make the specs pass:
```
const basePathRegex = new RegExp(REGEX);
```

Reply:
Yeah I just noted, it was a PR on the fly so I didn't clone the project and tested the lint by myself.
Edit incoming